internal/addrs/output_value.go

```diff
@@ -16,17 +16,28 @@ import (
 // that is defining it.
 //
 // This is related to but separate from ModuleCallOutput, which represents
-// a module output from the perspective of its parent module. Since output
-// values cannot be represented from the module where they are defined,
-// OutputValue is not Referenceable, while ModuleCallOutput is.
+// a module output from the perspective of its parent module. Outputs are
+// referencable from the testing scope, in general terraform operation users
+// will be referencing ModuleCallOutput.
 type OutputValue struct {
+	referenceable
```

Comment:
I cannot find any cases of this, but something to look out for is code which is meant to _exclude_ things like outputs by checking if the address is referenceable.